### PR TITLE
Add Z-Wave device info to OZW device pages

### PR DESCRIPTION
**Pull request by cgarwood**
## Proposed change

Depends on #6507 

Adds some basic Z-Wave device information to the Devices page for OZW devices. Based on code from the ZHA device pages.

image

Backend PR has been merged: https://github.com/home-assistant/core/pull/38265

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Checklist

- [x] The code change is tested and works locally.
- [x] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

### CODE REVIEW

File: src/panels/config/devices/device-detail/integration-elements/ozw/ha-device-info-ozw.ts [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+  @property({ attribute: false }) public hass!: HomeAssistant;
+
+  @property() public device!: DeviceRegistryEntry;
+
+  @internalProperty() private _ozwDevice?: OZWDevice;
+
+  protected updated(changedProperties: PropertyValues) {
+    if (changedProperties.has("device")) {
+      const ozwConnection = this.device.identifiers.find(
+        (conn) => conn[0] === "ozw"
```
Comment (bramkragten) [inline src/panels/config/devices/device-detail/integration-elements/ozw/ha-device-info-ozw.ts:28]:
```suggestion
        (identifier) => identifier[0] === "ozw"
```

File: src/panels/config/devices/device-detail/integration-elements/ozw/ha-device-info-ozw.ts [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+export class HaDeviceInfoOzw extends LitElement {
+  @property({ attribute: false }) public hass!: HomeAssistant;
+
+  @property() public device!: DeviceRegistryEntry;
+
+  @internalProperty() private _ozwDevice?: OZWDevice;
+
+  protected updated(changedProperties: PropertyValues) {
+    if (changedProperties.has("device")) {
+      const ozwConnection = this.device.identifiers.find(
```
Comment (bramkragten) [inline src/panels/config/devices/device-detail/integration-elements/ozw/ha-device-info-ozw.ts:27]:
```suggestion
      const ozwIndentifier = this.device.identifiers.find(
```

File: src/panels/config/devices/device-detail/integration-elements/ozw/ha-device-info-ozw.ts [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+        (conn) => conn[0] === "ozw"
+      );
+      if (!ozwConnection) {
```
Comment (bramkragten) [inline src/panels/config/devices/device-detail/integration-elements/ozw/ha-device-info-ozw.ts:30]:
```suggestion
      if (!ozwIndentifier) {
```

File: src/panels/config/devices/device-detail/integration-elements/ozw/ha-device-info-ozw.ts [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+        return;
+      }
+      const identifiers = ozwConnection[1].split(".");
+      fetchOZWNodeStatus(this.hass, identifiers[0], identifiers[1]).then(
```
Comment (bramkragten) [inline src/panels/config/devices/device-detail/integration-elements/ozw/ha-device-info-ozw.ts:34]: Move all this logic and the fetch to an async function that uses `await`.

**Review by bramkragten — APPROVED** [review]
Looks good! Some small remarks.

**Comment by cgarwood** [comment]
Haven't really worked with async/await in TypeScript. I _think_ I made it work like you requested in the comment, but let me know if that's not right.

File: src/panels/config/devices/device-detail/integration-elements/ozw/ha-device-info-ozw.ts [hunk 5]
```diff
@@ -24,19 +24,23 @@ export class HaDeviceInfoOzw extends LitElement {
...
+    );
+    if (!ozwIdentifier) {
+      return;
     }
+    const identifiers = ozwIdentifier[1].split(".");
+    await fetchOZWNodeStatus(this.hass, identifiers[0], identifiers[1]).then(
+      (dev) => {
+        this._ozwDevice = dev;
+      }
+    );
```
Comment (bramkragten) [inline src/panels/config/devices/device-detail/integration-elements/ozw/ha-device-info-ozw.ts:43]:
```suggestion
    this._ozwDevice = await fetchOZWNodeStatus(this.hass, identifiers[0], identifiers[1]);
```

**Review by bramkragten — APPROVED** [review]

**Pull request merged**